### PR TITLE
Clarify that the default branch is excluded

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

Because GitHub unfortunately excludes the default branch from yours, and until someone fixes it, the documentation should at least be up front about it.

### What's being changed:

Yours is clarified to explain that for _unspecified reasons_ the default branch is excluded from yours.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by jsoref** [comment]
To be clear: I would much rather GitHub's behavior be changed to match the current documentation and this PR rejected than have this PR merged. But, until it's changed, I'll settle for accurate documentation.

**Comment by ramyaparimi** [comment]
@jsoref Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/github/administering-a-repository/managing-branches-in-your-repository/viewing-branches-in-your-repository.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ shortTitle: View branches
 {% data reusables.repositories.navigate-to-repo %}
 {% data reusables.repositories.navigate-to-branches %}
 3. Use the navigation at the top of the page to view specific lists of branches:
-    - **Your branches**: In repositories that you have push access to, the **Yours** view shows all branches that you’ve pushed to, with the most recent branches first.
+    - **Your branches**: In repositories that you have push access to, the **Yours** view shows all branches that you’ve pushed to (**except** the *default* branch), with the most recent branches first.
```
Comment (lecoursen) [inline content/github/administering-a-repository/managing-branches-in-your-repository/viewing-branches-in-your-repository.md:18]:
```suggestion
    - **Your branches**: In repositories that you have push access to, the **Yours** view shows all branches that you’ve pushed to, excluding the default branch, with the most recent branches first.
```

**Review by lecoursen — DISMISSED** [review]
Thanks @jsoref! I'll apply my change and merge now. ⚡

**Review by lecoursen — APPROVED** [review]

**Comment by jsoref** [comment]
@lecoursen: if you can get the implementation changed, that'd still be great 🙏

**Pull request merged**